Week3/Homework/Exercise-4-Shopping-at-the-supermarket.js

```diff
@@ -0,0 +1,42 @@
+function addToShoppingCart(groceryItem) {
+
+    // const myOrders = ["bananas", "milk"];
+
+    const myOrders = [];
+
+    for (let i = 0; i < groceryItem.length; i++) {
+
+        myOrders.push(groceryItem[i]);
+
+    }
+
+    // console.log(myOrders[1]);
+
+    if (myOrders.length > 3) {
+
+        myOrders.shift();
+        return "You bought " + myOrders + "!";
+
+    }
+    else {
+
+        return "You bought " + myOrders + "!";
+    }
```

Comment:
Ah. I think the question asked to create a shopping cart array/list which contained bananas and milk by default and then you call the `addToShoppingCart` function passing in the shopping cart along with the item to add to the cart. And if the cart size got greater than 3, you were supposed to remove the first item. Something like:

```
function addToShoppingCart(cart, groceryItem) {
  // logic goes here
}

const cart = ['bananas', 'milk'];
addToShoppingCart(cart, 'butter');
addToShoppingCart(cart, 'carrot');
// and so on
```